spec/models/event_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  let(:event) {
+    Event.new(title: 'Minsk.rb best',
```

Comment:
In general, why not to initialize it with `factory_girl`? I also see a potential of using a `faker` gem here.
What do you think, @alexshgov?

Reply:
Sure, FactoryGirl + Faker are great combination